rest_framework_simplejwt/backends.py

```diff
@@ -31,6 +33,11 @@ def encode(self, payload):
         """
         Returns an encoded token for the given payload dictionary.
         """
+        if self.audience:
```

Comment:
Can you change this to `if self.audience is None`?  I prefer not to use implicit bool conversion when I can avoid it.  Can you also make the same change for the check against `self.issues` down below.